docs/backend-api/resources/tracking/tracker/chat.md

```diff
@@ -65,14 +82,22 @@ Marks all incoming chat messages as read for all or for given user trackers.
 
 | name | description | type | format |
 | :------ | :------ | :----- | :----- |
-| trackers | Optional array of Ids of the tracker (aka “object_id”). Tracker must belong to authorized user and not be blocked. | array of int | [999199, 999919,...] |
+| trackers | Optional array of Ids of the tracker (aka “object_id”). Tracker must belong to authorized user and not be blocked | array of int | [999199, 999919,...] |
```

Comment:
For why you remove a dot?

Reply:
because I don't leave dots at the end of descriptions in tables.

Reply:
but will add them from now